drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.haml

```diff
@@ -0,0 +1,57 @@
+:scss
+  .user-permission-report td ul {
+    padding-left: 0;
+    margin-left: 1em;
+  }
+:ruby
+  title = _('User Permission Report')
+  content_for :title, title
+
+= render 'warehouse_reports/breadcrumbs', tab: 'operational'
+
+.o-page
+  .o-page__header
+    - @prompt = t '.search_prompt'
+
+    .o-page__controls
+      .o-page__search
+        = render 'search_form', tooltip: 'Search on a user\'s first name, last name or email address', prompt: '', aria: { label: 'User Search' }, data: {placement: 'bottom'}
```

Comment:
I feel like you're working around some of this in changing the search form partial.  I'd rather see you just pass in some text for the search in the `prompt` argument and not bother setting `@prompt` above.

Reply:
Oh, missed this one. So let me tackle that now.

Reply:
Thanks for offering sanity :-) I've just pushed a little tweak.